src/lib/Panel.svelte

```diff
@@ -113,6 +128,9 @@
 		color: var(--panel-color);
 		border-radius: var(--panel-radius);
 		padding: var(--panel-padding);
+    max-width: 640px;
```

Comment:
Where did this come from?

Reply:
Ben said: "Tablet text line length should max out at 640px" - Perhaps we need to check with him if that's for only that one breakpoint? I suspect that it's a readability thing that applies in general

Reply:
Gotcha. Let's check, because it might make it a bit tight on large desktop. Also let's convert it to rems, that's what the rest of the stylesheet is using